### PR TITLE
🐛 fixing missing translation

### PR DESCRIPTION
**Pull request by coliss86**
#502 shows that some translation are missing, here is a fix

### CODE REVIEW

File: src/main/java/com/marginallyclever/makelangelo/Makelangelo.java [hunk 1]
```diff
@@ -386,7 +386,7 @@ private void saveGCode() {
 			save.run(myTurtle, myPlotter, mainFrame);
 		} catch(Exception e) {
 			logger.error("Error while exporting the gcode", e);
-			JOptionPane.showMessageDialog(mainFrame, e.getLocalizedMessage(), Translator.get("Error"), JOptionPane.ERROR_MESSAGE);
+			JOptionPane.showMessageDialog(mainFrame, e.getLocalizedMessage(), Translator.get("SaveError"), JOptionPane.ERROR_MESSAGE);
```
Comment (i-make-robots) [inline src/main/java/com/marginallyclever/makelangelo/Makelangelo.java:389]: what about ErrorTitle?

File: src/main/java/com/marginallyclever/makelangelo/Makelangelo.java [hunk 2]
```diff
@@ -583,7 +583,7 @@ public void windowClosing(WindowEvent e) {
 			dialog.setVisible(true);
 		} catch(Exception e) {
 			logger.error("Error while loading the file {}", filename, e);
-			JOptionPane.showMessageDialog(mainFrame, e.getLocalizedMessage(), Translator.get("Error"), JOptionPane.ERROR_MESSAGE);
+			JOptionPane.showMessageDialog(mainFrame, e.getLocalizedMessage(), Translator.get("LoadError"), JOptionPane.ERROR_MESSAGE);
```
Comment (i-make-robots) [inline src/main/java/com/marginallyclever/makelangelo/Makelangelo.java:586]: what about ErrorTitle?

File: src/main/java/com/marginallyclever/makelangelo/Makelangelo.java [hunk 3]
```diff
@@ -979,7 +979,7 @@ private void saveFile() {
 			saveDialog.run(myTurtle, mainFrame);
 		} catch(Exception e) {
 			logger.error("Error while saving the vector file", e);
-			JOptionPane.showMessageDialog(mainFrame, e.getLocalizedMessage(), Translator.get("Error"), JOptionPane.ERROR_MESSAGE);
+			JOptionPane.showMessageDialog(mainFrame, e.getLocalizedMessage(), Translator.get("SaveError"), JOptionPane.ERROR_MESSAGE);
```
Comment (i-make-robots) [inline src/main/java/com/marginallyclever/makelangelo/Makelangelo.java:982]: what about ErrorTitle?

**Comment by coliss86** [comment]
> why is this a separate PR from 502, which is still in draft?

the PR 502 detected some missing translation but the aim of 502 is not fixing the translation but adding new tests, so they are separated

File: src/main/java/com/marginallyclever/makelangelo/Makelangelo.java (hunk 1 above)
Comment (coliss86) [reply src/main/java/com/marginallyclever/makelangelo/Makelangelo.java]: Good point, I have replaced it with:
```
JOptionPane.showMessageDialog(mainFrame, Translator.get("SaveError") + e.getLocalizedMessage(), Translator.get("ErrorTitle"), JOptionPane.ERROR_MESSAGE);
```

**Pull request merged**